### PR TITLE
PIP-1095 atac template

### PR DESCRIPTION
**Pull request by paul-sud**

### CODE REVIEW

File: accession_steps/atac_steps.json [hunk 1]
```diff
@@ -0,0 +1,587 @@
...
+      "dcc_step_run": "atac-seq-signal-generation-step-v-1",
+      "dcc_step_version": "/analysis-step-versions/atac-seq-signal-generation-step-v-1-0/",
+      "wdl_files": [
+        {
+          "derived_from_files": [
+            {
+              "derived_from_filekey": "nodup_bam",
+              "derived_from_task": "filter",
+              "disallow_tasks": [
+                "choose_ctl"
```
Comment (leepc12) [inline accession_steps/atac_steps.json]: This is redundant. ATAC-seq pipeline doesn't have controls and `choose_ctl` task.

**Review by leepc12 — CHANGES_REQUESTED** [review]
I left one comment about controls.
Other than that it looks good to me.

File: accession_steps/jsonnet/atac_chip.jsonnet [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+linting afterwards `tox -e lint` to pretty-format the JSON, which will make the diffs
+more legible.
+*/
+{
+  local MAYBE_PREFERRED_DEFAULT = 'maybe_preferred_default',
+  local BedBigwigDerivedFromFiles(is_atac=false) = [
+    {
+      derived_from_filekey: 'nodup_bam',
+      derived_from_task: 'filter',
+      disallow_tasks: [
```
Comment (leepc12) [inline accession_steps/jsonnet/atac_chip.jsonnet:27]: Need to remove choose_ctl if `is_atac==True`.

Comment (paul-sud) [reply accession_steps/jsonnet/atac_chip.jsonnet:27]: yup fixed. Thanks

File: accession_steps/atac_steps.json (hunk 1 above)
Comment (paul-sud) [reply accession_steps/atac_steps.json]: Good catch, fixed this.

**Review by leepc12 — APPROVED** [review]
Looks good.

**Pull request merged**